spec/ruby/optional/capi/kernel_spec.rb

```diff
@@ -218,6 +232,34 @@
     end
   end
 
+  describe "rb_protect" do
+    it "will run a function with an argument" do
+      proof = [] # Hold proof of work performed after the yield.
+      res = @s.rb_protect_yield(7, proof) { |x| x + 1 }
+      res.should == 8
+      proof[0].should == 23
+    end
+
+    it "will allow cleanup code to return after break" do
+      proof = [] # Hold proof of work performed after the yield.
+      @s.rb_protect_yield(7, proof) { |x| break}
+      proof[0].should == 23
+    end
+
+    it "will allow cleanup code to return after break with value" do
+      proof = [] # Hold proof of work performed after the yield.
+      res = @s.rb_protect_yield(7, proof) { |x| break x + 1}
+      res.should == 8
+      proof[0].should == 23
+    end
+
+    it "will allow cleanup code to return after a raise" do
+      proof = [] # Hold proof of work performed after the yield.
+      lambda{ @s.rb_protect_yield(7, proof) { |x| raise NameError} }.should raise_error(NameError)
```

Comment:
Can you split the `lambda {` and `}.should raise_error(NameError)` on separate lines?